install.sh

```diff
@@ -0,0 +1,189 @@
+
+#!/usr/bin/env sh
+
+set -eu
+
+if [ -n "${GITHUB_ACTIONS-}" ]; then
+  set -x
+fi
+
+# Check pipefail support in a subshell, ignore if unsupported
+# shellcheck disable=SC3040
+(set -o pipefail 2> /dev/null) && set -o pipefail
+
+help() {
+  cat <<'EOF'
+Install a binary released on GitHub
+
+USAGE:
+    install.sh [options]
+
+FLAGS:
+    -h, --help      Display this message
+    -f, --force     Force overwriting an existing binary
+
+OPTIONS:
+    --repo REPO     Github Repository to install the binary from
+    --bin BIN       Name of the binary to install
+    --tag TAG       Tag (version) of the bin to install, defaults to latest release
+    --to LOCATION   Where to install the binary [default: ~/.cargo/bin]
```

Comment:
I don't think that `~/.cargo/bin` is a good default install location. If Rust/cargo is set up on the machine, users can use `cargo install` to install dora. Putting the binary in the cargo folder without going through cargo can lead to confusion (e.g. why does `cargo uninstall` not work for this binary?). Also, we're later adding `.cargo/bin` to the `PATH`, which probably also add lots of other binaries to the PATH (assuming the user didn't add `.cargo/bin` to the PATH before).

If Rust/Cargo is not installed, it might be weird to still have a `.cargo/bin` folder. It also might lead to issues when the user decides to install `rustup` later (does it complain if the dir already exists?).

Reply:
That makes sense. I'm going to change it to something like `~/.dora/bin`